mbl/urls.py

```diff
@@ -36,6 +36,9 @@
     url(r'^person/(?P<person_id>[0-9]+)/investigator/add/$',views.add_investigator_record,name='add-investigator'),
     url(r'^person/(?P<person_id>[0-9]+)/investigator/(?P<research_id>[0-9]+)/edit/$',views.edit_investigator_record,name='edit-investigator'),
     url(r'^person/(?P<person_id>[0-9]+)/investigator/(?P<research_id>[0-9]+)/delete/$', views.delete_investigator_record, name='delete-investigator'),
+    url(r'^person/(?P<person_id>[0-9]+)/position/add/$', views.position, {'position_id': None}, name='add-position'),
+    url(r'^person/(?P<person_id>[0-9]+)/position/(?P<position_id>[0-9]+)/edit/$', views.position, name='edit-position'),
```

Comment:
since add and edit both point to the same view function, why two urls?

Reply:
I was not able to pass a none object from the html template to the django view. So, I am passing the none object from the urls file itself. That is why I have used an additional url here.

Reply:
Why not different view functions? You won't need to add/edit in the same function based on `position_id` being `None`.

Reply:
@hashknot Earlier there were two views to handle these cases however, there was a lot of redundant code between them. That's why only one function is being used to add and edit.